readme.md

```diff
@@ -61,6 +61,15 @@ Default: `Infinity`
 
 Milliseconds to wait before stopping the search and resolving the promise to `undefined`.
 
+##### expectEntireElement
+
+Type: `boolean`<br>
```

Comment:
```suggestion
Type: `boolean`\
```

Reply:
Want me to change it everywhere in this file in this PR?

Reply:
Sure